common/pb/serverinfo_card.proto

```diff
@@ -4,6 +4,7 @@ import "serverinfo_cardcounter.proto";
 message ServerInfo_Card {
     optional sint32 id = 1 [default = -1];
     optional string name = 2;
+    optional string uuid = 17;
```

Comment:
to me it'd still make the most sense to put this field in sequence, that way we don't mess things up if we'd try to add another field in a future

Reply:
I have no objections to this.